sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/aio/_router_administration_client_async.py

```diff
@@ -0,0 +1,1427 @@
+    StaticRule,
+    ExpressionRule,
+    FunctionRule,
+)
+from .._models import (
+    JobQueue,
+    JobQueueItem,
+)
+
+from .._version import SDK_MONIKER
+
+_SERIALIZER = Serializer()
+_SERIALIZER.client_side_validation = False
+
+
+class RouterAdministrationClient(object):  # pylint: disable=client-accepts-api-version-keyword,too-many-public-methods,too-many-lines
+    """A client to interact with the AzureCommunicationService JobRouter service.
+
+    This client provides operations to create, update, list and delete the following entities: classification policy,
+    exception policy, distribution policy and queue.
+
+    :param str endpoint:
+        The endpoint of the Azure Communication resource.
+    :param CommunicationTokenCredential credential:
+        The credentials with which to authenticate
+    """
+
+    def __init__(
+            self,
+            endpoint: str,
+            credential: CommunicationTokenCredential,
+            **kwargs: Any
+    ) -> None:
+        # type: (...) -> None
+        if not credential:
+            raise ValueError("credential can not be None")
+
+        try:
+            if not endpoint.lower().startswith('http'):
+                endpoint = "https://" + endpoint
+        except AttributeError:
+            raise ValueError("Host URL must be a string")
+
+        parsed_url = urlparse(endpoint.rstrip('/'))
+        if not parsed_url.netloc:
+            raise ValueError("Invalid URL: {}".format(endpoint))
+
+        self._endpoint = endpoint
+        self._authentication_policy = get_authentication_policy(endpoint, credential)
+        self._client = AzureCommunicationJobRouterService(
+            self._endpoint,
+            authentication_policy=self._authentication_policy,
+            sdk_moniker=SDK_MONIKER,
+            **kwargs)
+
+    @classmethod
+    def from_connection_string(
+            cls,
+            conn_str: str,
+            **kwargs: Any
+    ) -> "RouterAdministrationClient":
+        # type: (...) -> RouterAdministrationClient
+        """Create RouterClient from a Connection String.
+
+        :param str conn_str:
+            A connection string to an Azure Communication Service resource.
+        :return: Instance of RouterAdministrationClient.
+        :rtype: ~azure.communication.jobrouter.aio.RouterAdministrationClient
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_authentication_async.py
+                :start-after: [START admin_auth_from_connection_string_async]
+                :end-before: [END admin_auth_from_connection_string_async]
+                :language: python
+                :dedent: 8
+                :caption: Authenticating a RouterAdministrationClient from a connection_string
+        """
+        endpoint, access_key = parse_connection_str(conn_str)
+
+        return cls(endpoint, access_key, **kwargs)
+
+    # region ExceptionPolicyAio
+
+    @overload
+    def create_exception_policy(
```

Comment:
While we should remove the overloads for the `create_` operations - overloads for async operations should also be async.

Reply:
I have removed the overloads for `create_` methods